### PR TITLE
Add waitReplicas to Pipeline interface

### PR DESCRIPTION
**Pull request by gkorland**

### CODE REVIEW

**Review by sazzad16 — APPROVED** [review]

**Review by sazzad16 — CHANGES_REQUESTED** [review]
@gkorland Where is `waitReplicas` test?

**Comment by gkorland** [comment]
@sazzad16 you're right but since we don't have any cluster/replication here I thought it will be useless, but I added a waitReplicas with 0 replicas just to make sure we still cover this code

**Comment by sazzad16** [comment]
@gkorland yeah, I just realized, it's not Jedis test environment.

File: src/main/java/com/redislabs/redisgraph/RedisGraphPipeline.java [hunk 1]
```diff
@@ -57,17 +57,6 @@
...
-    /**
-     * Execute a Cypher query with arguments
-     * @param graphId a graph to perform the query on
-     * @param query Cypher query
-     * @param args
-     * @return a response which builds the result set with the query answer.
-     * @deprecated use {@link #query(String, String, Map)} instead.
-     */
-    @Deprecated
-    Response<ResultSet> query(String graphId, String query, Object ...args);
```
Comment (sazzad16) [inline src/main/java/com/redislabs/redisgraph/RedisGraphPipeline.java:69]: Removing this means we'll have to target this PR for next major release. I don't think next major release is near.

**Review by sazzad16 — CHANGES_REQUESTED** [review]

File: src/main/java/com/redislabs/redisgraph/RedisGraphPipeline.java (hunk 1 above)
Comment (gkorland) [reply src/main/java/com/redislabs/redisgraph/RedisGraphPipeline.java:69]: I shouldn't have add it from the beginning it was a mistake to add it to this new class when I copied it from the Transaction class.

**Review by DvirDukhan — APPROVED** [review]

**Pull request merged**